### ISSUE TITLE
Release Twisted 13.1

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#6575]
|[<img alt="ashfall's avatar" src="https://avatars.githubusercontent.com/u/1138873?s=50" width="50" height="50">](https://github.com/ashfall)| @ashfall reported|
|-|-|
|Trac ID|trac#6575|
|Type|task|
|Created|2013-06-14 17:57:24Z|
|Branch|https://github.com/twisted/twisted/tree/releases/release-13.1.0-6575|

It is time.

<details><summary>Searchable metadata</summary>

```
trac-id__6575 6575
type__task task
reporter__ashfall ashfall
priority__normal normal
milestone__Twisted_13_1_0 Twisted-13.1.0
branch__releases_release_13_1_0_6575 releases/release-13.1.0-6575
branch_author__ 
status__closed closed
resolution__fixed fixed
component__release_management release management
keywords__ 
time__1371232644000000 1371232644000000
changetime__1372614133000000 1372614133000000
version__None None
owner__ashfall ashfall
cc__radix cc__thijs
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_3"></a>|@thijstriemstra commented|
|-|-|

There's a incorrect news file in [twisted/topfiles/](twisted/topfiles/) that should be removed as well (perhaps a ticket should be filed that this doesn't happen again although not sure how to solve that).

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_4"></a>|@exarkun commented|
|-|-|

> There's a incorrect news file in trunk/twisted/topfiles/ that should be removed as well

That file being...?

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_5"></a>|@thijstriemstra commented|
|-|-|

Replying to [exarkun](#note_4):
> That file being...?

https://twistedmatrix.com/trac/browser/branches/releases/release-13.1.0-6575/twisted/topfiles/6510.imsc

**Comment by twisted-trac** [comment]
|[<img alt="ashfall's avatar" src="https://avatars.githubusercontent.com/u/1138873?s=50" width="50" height="50">](https://github.com/ashfall)<a name="note_6"></a>|@ashfall commented|
|-|-|

(In [38892]) Manually delete an incorrect news file.

refs [#6575](https://github.com/twisted/twisted/issues/6575)

**Comment by twisted-trac** [comment]
|[<img alt="ashfall's avatar" src="https://avatars.githubusercontent.com/u/1138873?s=50" width="50" height="50">](https://github.com/ashfall)<a name="note_7"></a>|@ashfall removed owner|
|-|-|

Please review.

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)|@exarkun set owner to @exarkun<br>@exarkun set status to `assigned`|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_9"></a>|@exarkun set owner to @ashfall<br>@exarkun set status to `new`|
|-|-|

Thanks!  Here's some points, mostly extremely nit-picky and about text written by other people, for your enjoyment:

  1. The news item for [#6291](https://github.com/twisted/twisted/issues/6291) talks about `SyncTestCase`, but the class is `SynchronousTestCase`
  1. The news item for [#6350](https://github.com/twisted/twisted/issues/6350) doesn't follow the [style guide](https://twistedmatrix.com/trac/wiki/ReviewProcess#Newsfiles)
  1. Neither does the item for [#6380](https://github.com/twisted/twisted/issues/6380)
  1. Neither does the item for [#6390](https://github.com/twisted/twisted/issues/6390)
  1. The item for [#6521](https://github.com/twisted/twisted/issues/6521) has a linebreak in a hilarious place.  Maybe rephrasing it so the method doesn't have to be part of the same word as the class would help?
  1. The news item for [#6407](https://github.com/twisted/twisted/issues/6407) is probably overly optimistic.  I suspect what it means is that it contains *fewer* spelling mistakes than it used to.
  1. The item for [#5847](https://github.com/twisted/twisted/issues/5847) is confusingly worded.  I would say "; the epoll reactor now uses the stdlib-provided epoll support."
  1. The formatting for the [#6514](https://github.com/twisted/twisted/issues/6514) item is all messed up.
  1. The item for [#6532](https://github.com/twisted/twisted/issues/6532) is perhaps questionable.  I guess erring on the side of announcing more features is good.  This one strikes me as a little weird because it's describing a feature of another feature that was introduced a couple lines above.  I suppose it's fine though.
  1. The item for [#5386](https://github.com/twisted/twisted/issues/5386) uses a partially qualified name.  Fully qualified is better.
  1. Same for [#4151](https://github.com/twisted/twisted/issues/4151).
  1. The item for [#5435](https://github.com/twisted/twisted/issues/5435) is strangely specific: it talks about a new attribute of a new interface, but it doesn't bother mentioning that the interface itself is new.
  1. The item for [#2980](https://github.com/twisted/twisted/issues/2980) has some weird extra whitespace (that seems like it ought to have been stripped by the tools).
  1. There is a conflict in `Twisted.Quotes`, as I suppose there always is now.  ReleaseProcess doesn't seem to talk about this, though - it might be nice to add some steps to that page to explain how to resolve this (as it differs from the normal conflict resolution).

Don't forget that most news items appear in two different NEWS files.  Also, I think it might be helpful to communicate these changes to the original authors somehow, or to improve ReviewProcess#Newsfiles somehow (although some of the points are clearly already addressed in that documentation).  Developers should be writing these things correctly so minimal changes are required at release time.

Thanks again!  Please merge this as soon as you're happy with the news files.

**Comment by twisted-trac** [comment]
|[<img alt="ashfall's avatar" src="https://avatars.githubusercontent.com/u/1138873?s=50" width="50" height="50">](https://github.com/ashfall)<a name="note_10"></a>|@ashfall set status to `closed`|
|-|-|

(In [39001]) Merge releases/release-13.1.0-6575

Author:ashfall
Reviewer: exarkun
Fixes: [#6575](https://github.com/twisted/twisted/issues/6575)

Release Twisted 13.1.0
